.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: c
```

Comment:
while it is not needed; since there is no way for other developers to test their changes in s390x unless they use the IBM cloud it will be an important addition (as discussed in #89 ).  i'll remove it from the initial port PR and keep it on my own tree, but then it will be only effective after rebasing on top of your master and might be too late to prevent breaking changes being introduced.

it will required that you "install" travis into your github account though but there is no cost to do so if using their opensource cloud.

additionally it could help filter out PR that could otherwise break in architectures that you are not normally using like macOS and windows or some non x86 linux if extended further and with further tweaks, which I can assist with.